src/main/groovy/io/openliberty/tools/gradle/tasks/DevTask.groovy

```diff
@@ -207,13 +207,13 @@ class DevTask extends AbstractServerTask {
         DevTaskUtil(File serverDirectory, File sourceDirectory, File testSourceDirectory,
                     File configDirectory, File projectDirectory, List<File> resourceDirs,
                     boolean  hotTests, boolean  skipTests, String artifactId, int serverStartTimeout,
-                    int verifyAppStartTimeout, int appUpdateTimeout, double compileWait, 
+                    int verifyAppStartTimeout, int appUpdateTimeout, double compileWait,
                     boolean libertyDebug, boolean pollingTest, boolean container, File dockerfile
         ) throws IOException {
             super(serverDirectory, sourceDirectory, testSourceDirectory, configDirectory, projectDirectory,
                     resourceDirs, hotTests, skipTests, false, false, artifactId,  serverStartTimeout,
-                    verifyAppStartTimeout, appUpdateTimeout, ((long) (compileWait * 1000L)), libertyDebug, 
-                    true, true, pollingTest, container, dockerfile);
+                    verifyAppStartTimeout, appUpdateTimeout, ((long) (compileWait * 1000L)), libertyDebug,
+                    true, true, pollingTest.booleanValue(), container.booleanValue(), dockerfile);
```

Comment:
Why does this need .booleanValue()?  pollingTest and container are primitive booleans here

Reply:
I made this mistake too. I thought we wanted the class variable in all cases but in this case we need to refer to the parameter of the DevTaskUtil constructor.

Reply:
The `.booleanValue()` additions were changes pulled into my branch. Do you want them removed?